library/ssl_misc.h

```diff
@@ -20,6 +20,9 @@
 
 #include "psa/crypto.h"
 #include "psa_util_internal.h"
+#if defined(MBEDTLS_USE_PSA_CRYPTO) || defined(MBEDTLS_SSL_PROTO_TLS1_3)
```

Comment:
Please note that MBEDTLS_USE_PSA_CRYPTO was recently removed from the `development` branch as it's assumed to be always set (https://github.com/Mbed-TLS/mbedtls/pull/9923). Therefore I think we can just remove the guard here.